pygfx/renderers/wgpu/_shaderlib.py

```diff
@@ -0,0 +1,693 @@
+# Some guidelines:
+#
+# Put code in logical blocks using methods. That way they can be reused
+# and also show up in the "source structure" of most IDE's.
+#
+# Colors should be converted to the physical colorspace in the fragment
+# shader (because you generally want to interolate in srgb). Other than
+# that, the color should be turned to physical using srgb2physical()
+# as soon as reasonably possible. For clarity it helps to add a "_srgb"
+# suffix to colors not yet converted. Most colors on a uniform can also
+# be considered srgb.
+#
+# That pi scale factor for the lights is related to artist-friendly colors or something.
+# Please let me know if you know more about this :)
+
+
+class Shaderlib:
+    def light_deps_basic(self):
+        return """
+        fn getDistanceAttenuation(light_distance: f32, cutoff_distance: f32, decay_exponent: f32) -> f32 {
+            if ( cutoff_distance > 0.0 && decay_exponent > 0.0 ) {
+                return pow( clamp( - light_distance / cutoff_distance + 1.0, 0.0, 1.0), decay_exponent );
+            }
+            return 1.0;
+        }
```

Comment:
@panxinmiao are these wgsl snippets translated from glsl in TheeJS? It looks like this snippet is from the non-physically-correct part: https://github.com/mrdoob/three.js/blob/a3a4c821229ac554f3ce53cc2b9460cfd205aaaa/src/renderers/shaders/ShaderChunk/lights_pars_begin.glsl.js

I think we'd want to implement things as if `PHYSICALLY_CORRECT_LIGHTS` is always True.

Reply:
Mmm, it looks like this is the only snippet where the `PHYSICALLY_CORRECT_LIGHTS` applies. So easy enough to fix :)